src/assets/src/components/queue.tsx

```diff
@@ -573,6 +574,7 @@ export function QueuePage(props: PageProps) {
             onChangeBackend={setSelectedBackend}/>
     return (
         <div>
+            <HelmetTitle title={queue?.name ?? queue_id.toString()} />
```

Comment:
should this be `Queue SI 240`? if somebody created a queue as `SI 240` currently it just says ` `SI 240 ROHQ`. This is optional